modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -126,7 +126,8 @@ Configuration example:
 </scopeLoggingConfigurations>
 ----
 | No
-| `target` | The CloudHub 2.0 target name where to deploy the app. | Yes
+| `target` | The CloudHub 2.0 target name where to deploy the app. + 
```

Comment:
The CloudHub 2.0 target name to deploy the app to. 
or 
The CloudHub 2.0 target name to which to deploy the app to.